src/GitExtensions.Extensibility.Tools/tools/Download-GitExtensions.ps1

```diff
@@ -0,0 +1,194 @@
+param(
+  [Parameter(Mandatory=$true)]
+  [string] $ExtractRootPath, 
+  [Parameter(Mandatory=$true)]
+  [string] $Version, 
+  [ValidateSet('GitHub','AppVeyor', ignorecase=$False)]
+  [string] $Source = "GitHub")
+
+function Test-LocalCopy 
+{
+	Param(
+		[Parameter(Mandatory=$true, Position=0)]
+		[string] $ExtractPath,
+		[Parameter(Mandatory=$true, Position=1)]
+		[string] $FileName
+	)
+
+	$FilePath = [System.IO.Path]::Combine($ExtractPath, $FileName);
+	if (Test-Path $FilePath)
+	{
+		Write-Host "Download '$FileName' already exists.";
+		return $true;
+	}
+	
+	return $false;
+}
+
+function Get-GitExtensionsApp
+{
+    Param(
+      [Parameter(Mandatory=$true, Position=0)]
+      [string] $Source,
+      [Parameter(Mandatory=$true, Position=1)]
+      [string] $Version
+    )
+    
+    if ($Source -eq 'GitHub') {
+        return DownloadFromGitHub -Version $Version
+    }
+    elseif ($Source -eq 'AppVeyor') {
+        return DownloadFromAppVeyor -Version $Version

```

Comment:
I can't seem to find these method...
I suspect you may have missed these during a refactor.

Reply:
Whole this function somehow appeared here... not used anywhere...